### PR TITLE
Added Aave connector on flashloan

### PR DESCRIPTION
**Pull request by thrilok209**

### CODE REVIEW

File: contracts/flashloan/Instapool/main.sol [hunk 1]
```diff
@@ -138,7 +143,7 @@ contract DydxFlashloaner is Helper, ICallee, DydxFlashloanBase {
             require(sub(initialBal, finalBal) <= _dif, "eth-amount-paid-less");
         } else {
             uint256 _decimals = TokenInterface(token).decimals();
-            uint _dif = wmul(convertTo18(_amount, _decimals), 10000000000); // Taking margin of 0.00000001%
+            uint _dif = wmul(convertTo18(_amount, _decimals), 200000000000); // Taking margin of 0.0000002%
```
Comment (KaymasJain) [inline contracts/flashloan/Instapool/main.sol:146]: Do you think Aave has the decimals problem as the 1 atoken is always equals to 1 token.

**Review by KaymasJain — APPROVED** [review]

**Comment by KaymasJain** [comment]
I've not checked the Aave connector code as it seems to be copy-paste. Although, you sure you used Aave v2 addresses as old flashloan had Aave v1 I guess (not sure)

File: contracts/flashloan/Instapool/main.sol (hunk 1 above)
Comment (thrilok209) [reply contracts/flashloan/Instapool/main.sol:146]: We shouldn't have any problem due to that. I copied this logic from the old contract. I think it was for the debt fee on Aave.

**Comment by thrilok209** [comment]
> I've not checked the Aave connector code as it seems to be copy-paste. Although, you sure you used Aave v2 addresses as old flashloan had Aave v1 I guess (not sure)

Yess. I used the aaveV2 connector from here: https://github.com/Instadapp/dsa-connectors/blob/main/contracts/mainnet/connectors_old/aave_v2.sol

File: contracts/flashloan/Instapool/main.sol (hunk 1 above)
Comment (KaymasJain) [reply contracts/flashloan/Instapool/main.sol:146]: So no need to change anything on this? Should we proceed with the merge then? Also, I think there's is no issue on the borrow fee on Aave v2. Right?

Comment (thrilok209) [reply contracts/flashloan/Instapool/main.sol:146]: Yeah. we can merge this

**Pull request merged**